### DISCUSSION TITLE
Unlimited users

### DISCUSSION THREAD

**Discussion by ghost** [strapi/strapi#6101] [Legacy Topics]
Hey everybody,

With Strapi, can I create unlimited users. I own a huge writers community + 1000 writers. I want to use strapi to create custom UI to manage my content with +1000 writers.

Thanks.

**Comment by sunnysonx** [comment]
@olivaxa2 Hello,
Of course, you can.

**Reply by ghost** [reply]
👍🏻 Cool. Strapi Competitors limit this feature to 10 users. +15$ per users...

**Reply by sunnysonx** [reply]
Yeah, but Strapi is an open-source project, so even if it had user limitations you could just modify them per your own project.

**Comment by robertpiosik** [comment]
@olivaxa2 Strapi team is working on [User roles and permissions](https://portal.productboard.com/strapi/1-public-roadmap/c/8-users-roles-permissions-ce-ee) and it will be something you would have to pay for. I believe that sooner or later community will come with a free alternative to this paid extension.

**Reply by filipesmedeiros** [reply]
Doesn't this already exist in the free plug-in included with the base build?

**Reply by robertpiosik** [reply]
It works on API level, not admin dashboard.

**Reply by ghost** [reply]
> Community Edition (CE) will be limited to 3 pre-defined roles (Administrators, Author)

Authors are just « writers » so I think free CE Edition is ok for me.

**Reply by robertpiosik** [reply]
@olivaxa2 Good to hear that! :)
